cub/cub/util_type.cuh

```diff
@@ -43,12 +43,8 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cfloat>
-#include <iostream>
-#include <iterator>
-#include <limits>
-
-#include <cuda.h>
+#include <cuda/std/limits>
+#include <cuda/__cccl_config> // _LIBCUDACXX_CUDACC_VER
```

Comment:
That should not be necessary as we pull in `__cccl_config` from cuda/__config

Reply:
@miscco did you mean `cub/config.cuh`?

Reply:
I meant that `<cuda/std/limits>` already pulls it in but also `<cub/config.cuh>`

Reply:
I prefer having independent headers with explicit inclusion of what's used. Do you anticipate any issues with including `cuda/__cccl_config` directly?